pandas/core/generic.py

```diff
@@ -908,23 +889,7 @@ def squeeze(self, axis=None):
         ]
 
     def swaplevel(self: FrameOrSeries, i=-2, j=-1, axis=0) -> FrameOrSeries:
-        """
-        Swap levels i and j in a MultiIndex on a particular axis
-
-        Parameters
-        ----------
-        i, j : int, str (can be mixed)
-            Level of index to be swapped. Can pass level name as string.
-
-        Returns
-        -------
-        swapped : same type as caller (new object)
-        """
-        axis = self._get_axis_number(axis)
-        result = self.copy()
-        labels = result._data.axes[axis]
-        result._data.set_axis(axis, labels.swaplevel(i, j))
-        return result
+        raise AbstractMethodError(self)
```

Comment:
Rationale?

Reply:
is it because of the additional copy parameter in Series.swaplevel that the generic method cannot retained and frame and series call super in preference to removing the generic implementation.

Reply:
can this be removed entirely?